pyiode/pyiode_exec.pyx

```diff
@@ -6,18 +6,109 @@
 #  IODE identity execution
 #  -----------------------
 # 
-#  TODO
-#  ---- 
-#   idt_execute(idt_list: Union(str, List[str]), sample: Union(str, List[str]), var_files: Union(str, List[str]), scl_files: Union(str, List[str])) | Execute a list of identities on a given sample
+#   idt_execute(sample: Optional[Union[str, List[str]]] = None, idt_list: Optional[Union[str, List[str]]] = None, var_files: Optional[Union[str, List[str]]] = None, scl_files: Optional[Union[str, List[str]]] = None,trace: int = 0) | Execute a list of identities on a given sample
 
+cdef extern from "iode.h": 
+    cdef int    IodeExecuteIdts(char *smpl, char *idt_list, char *var_files, char *scl_files, int trace)
 
-def idt_execute(idt_list: Union[str, List[str]], 
-                sample: Union[str, List[str]], 
-                var_files: Union[str, List[str]], 
-                scl_files: Union[str, List[str]]):
-    '''
-    Estimate a (block of) equations
-     
+def idt_execute(sample: Optional[Union[str, List[str]]] = None, 
+                idt_list: Optional[Union[str, List[str]]] = None, 
+                var_files: Optional[Union[str, List[str]]] = None, 
+                scl_files: Optional[Union[str, List[str]]] = None,
+                trace: bool = False):
+    r'''
+    Execute a list of identities
+    
+    Parameters
+    ----------
+    sample: Optional[Union[str, List[str]]] = None
+            range of period on which the identities must be calculated
+            if sample is None or empty, the WS sample is used 
+            
+    idt_list: Optional[Union[str, List[str]]] = None
+            list of identities to execute
+            if empty, all identities are executed
+            
+    var_files: Optional[Union[str, List[str]]] = None
+            list of files the needed variables must be read from. "WS" means current current Variable WS.
+            if empty, only the current KV_WS is used
+            
+    scl_files: Optional[Union[str, List[str]]] = None
+            list of files the needed scalars must be read from. "WS" means current current WS.
+            if empty, only the current workspace is used
+                        
+    trace: int = 0
+            optional trace indicating the source of the variables and scalars. For example:
+            
+            Execution of identities
+            
+            Parameters
+                Execution sample : 1960Y1:2015Y1
+            
+                Variables loaded
+                    From WS : AOUC AOUC_ COTRES DEBT DTF DTH EFMY EFXY EX FLG FLGR GAP2 GAP_ GOSF HF  
+                    
+                Scalars loaded
+                    From ../data/fun : gamma gamma2 gamma3 gamma4 knf3
+            
+    Examples
+    --------
+        # define the "trace" file
```

Comment:
Please look at the Examples in the pyiode_sample.pyx file and rewrite the whole Examples section here

Reply:
Your Examples section here should start with 
```python
Examples
----------
>>> from iode import w_dest, ws_clear_var, ws_sample_set, ws_load_idt, idt_execute, ws_load_var, get_var, set_var
````

Reply:
TODO.

Reply:
If you want me to do it, please add a comment `TODO: (ALD) rewritte Examples section below` above the function and click on 'Resolve conversion' to each related conservation below

Reply:
Done